src/components/Schedule/TableView.tsx

```diff
@@ -1,15 +1,172 @@
+        prof: professors,
+        classSize: course.capacity,
+        startDate: meetingTime.startDate,
+        endDate: meetingTime.endDate,
+      } as Appointment;
+    });
+
+    return assignedCourses;
+  });
+  return appointments;
+};
+
+const populateTable = (courses: Appointment[]) => {
+  var table_b = document.getElementById("table_body");
+  if (table_b != null) {
+    //remove previous table
+    if (table_b.childNodes.length !== 0) {
+      while (table_b.childNodes.length > 0) {
+        table_b.childNodes[0].remove();
+      }
+    }
+    var nTd, nTr;
+    for (var i = 0; i < courses.length; i++) {
+      nTr = document.createElement("Tr");
+
+      var sdate = courses[i].startDate
+        .toString()
+        .split(" ")
+        .slice(0, 4)
+        .join(" ");
+      var stime = courses[i].startDate
+        .toString()
+        .split(" ")
+        .slice(4, 5)
+        .join(" ");
+      var edate = courses[i].endDate
+        .toString()
+        .split(" ")
+        .slice(0, 4)
+        .join(" ");
+      var etime = courses[i].endDate
+        .toString()
+        .split(" ")
+        .slice(4, 5)
+        .join(" ");
+
+      //Course title
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = courses[i].subject + " " + courses[i].courseTitle;
+      nTr.appendChild(nTd);
+
+      //Schedule time
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = stime + " / " + etime;
+      nTr.appendChild(nTd);
+
+      //Term
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = "Summer";
+      nTr.appendChild(nTd);
+
+      //Prof
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = courses[i].prof[0];
+      nTr.appendChild(nTd);
+
+      //Course Code
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = courses[i].section;
+      nTr.appendChild(nTd);
+
+      //Start/End Data
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = sdate + " / " + edate;
+      nTr.appendChild(nTd);
+
+      //# of students
+      nTd = document.createElement("Td");
+      nTd.style.padding = "8px 16px";
+      nTd.innerText = courses[i].classSize.toString();
+      nTr.appendChild(nTd);
+      table_b?.appendChild(nTr);
+    }
+  } else {
+    //table not loaded yet, try again in 100ms
+    setTimeout(function () {
+      populateTable(courses);
+    }, 100);
+  }
+};
+
+interface TableProps {
+  data: CourseSection[];
+}
+
+export const TableView = (props: TableProps) => {
+  const { data } = props;
+  const appointments = buildAppointments(data);
+  populateTable(appointments);
   return (
     <TableContainer overflowY="auto">
       <Table size="sm" variant="striped" colorScheme="gray">
```

Comment:
Should change the CRN header to something like Section since that's the data we're using there now